packages/go_router/example/lib/stream_listener_router.dart

```diff
@@ -0,0 +1,318 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I don't quite understand the purpose of this stream_listener_router in example, usually we use example to demonstrate how to use an API, and we use tests to avoid issue regression.  is this file added to avoid the regression for issue#150312 ?

Reply:
@chunhtai do you think we should keep stream_listener_router in the example?